### ISSUE TITLE
A11y: focus highlights are missing on 'create room' dialog

### ISSUE THREAD

**Issue by ara4n** [vector-im/element-web#19742]
### Steps to reproduce

1. Create a new room from + in the Rooms section of the RoomList
2. Tried to navigate the dialog using tab/shift-tab for keyboard shortcuts

#### What did you expect?

A visual focus outline should show me what controls are highlighted, so that if i press space/enter to interact with them (e.g. toggle e2ee or expand the 'show advanced' section or hit 'cancel' or 'create room' button) i know where i am and what i'm interacting with.

#### What happened instead?

The focus outline disappears.  It's shown around the Name, Topic & Private/Public room controls, but none of the bottom half of the dialog.  Screenshot shows the private/public select control being selected (obviously i can't show the unfocused stuff not being focused :P)

<img width="517" alt="Screenshot 2021-11-13 at 13 40 36" src="https://user-images.githubusercontent.com/1294269/141646127-9c63502f-a28d-4818-9dc0-04dc0c8caacd.png">

### Operating system

macOS 12.0.1 on M1

### Application version

nightly

### How did you install the app?

nightly

### Homeserver

localhost

### Will you send logs?

No

**Comment by yuktea** [comment]
I would like to try this out

**Comment by ilmari** [comment]
Buttons in other dialogs (e.g. report content, devtools) don't have visible focus state either.
